charts/mariadb/v7.3.14/app-readme.md

```diff
@@ -5,5 +5,5 @@
 MariaDB is developed as open source software and as a relational database it provides an SQL interface for accessing data. The latest versions of MariaDB also include GIS and JSON features.
 
 ## Notable changes
-### v6.7.4
+### v7.3.14
 **Warning:** Upgrade from previous version is currently not supported.
```

Comment:
If upgrade works, then this line should be removed. 

Is there anything else that was found in the release notes of 7.3.14?